src/main/java/org/opentripplanner/routing/edgetype/StreetEdgeTraversal.java

```diff
@@ -208,6 +213,25 @@ public StateEditor doTraverse(RoutingRequest options, TraverseMode traverseMode)
                 // TODO: we don't have this data yet
             }
 
+            OptionAttribute curbRamp = extraOptions.getOption(NihOption.CURB_RAMP);
+            if (curbRamp == CurbRamp.YES) {
+                // prefer audited edges with curb ramps
+                weight *= 0.9;
```

Comment:
I'm not sure how much of this is existing OTP style, but it might be a good idea to use an enum for these weights.